### DISCUSSION TITLE
Incompatibility Issue Between sqlmodel and pydantic-settings

### DISCUSSION THREAD

**Discussion by Kokoserver** [fastapi/sqlmodel#708] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
I am encountering an issue with the dependencies of `my project`. The error message is as follows:

Because no versions of sqlmodel match >0.0.12,<0.0.13
 and sqlmodel (0.0.12) depends on pydantic (>=1.9.0,<2.0.0), sqlmodel (>=0.0.12,<0.0.13) requires pydantic (>=1.9.0,<2.0.0).
And because pydantic-settings (2.1.0) depends on pydantic (>=2.3.0)
 and no versions of pydantic-settings match >2.1.0,<3.0.0, sqlmodel (>=0.0.12,<0.0.13) is incompatible with pydantic-settings (>=2.1.0,<3.0.0).
So, because `my project`  depends on both pydantic-settings (^2.1.0) and sqlmodel (^0.0.12), version solving failed.
```

### Description

i am using fastapi with sqlmodel and i need to use pydntic-settings to control my application configuration, but while trying to install that i get the above error

### Operating System

Windows

### Operating System Details

running on window 11

### SQLModel Version

sqlmodel = "^0.0.12"

### Python Version

python-3.11

**Comment by YuriiMotov** [comment, marked as answer]
There is no such problem in current version (0.0.24).
If you need to use Pydantic V1, you don't need to import `pydantic-settings`: https://docs.pydantic.dev/1.10/usage/settings/
